src/Illuminate/Mail/Mailer.php

```diff
@@ -160,7 +160,7 @@ public function bcc($users)
     }
 
     /**
-     * Send a new message with only an HTML part.
```

Comment:
This is not wrong. Is just a matter of how you pronounce HTML

Reply:
Seems valid, I'll remove that change.